playbooks/common/openshift-etcd/config.yml

```diff
@@ -53,15 +53,15 @@
         -C {{ etcd_generated_certs_dir }}/{{ item.etcd_cert_subdir }} .
     args:
       creates: "{{ etcd_generated_certs_dir }}/{{ item.etcd_cert_subdir }}.tgz"
-    with_items: etcd_needing_server_certs
+    with_items: "{{ etcd_needing_server_certs }}"
```

Comment:
Yeah! Still testing 1.9.4 with this -- looks like a lot of these need defaults when templated.